modules/percolator/src/main/java/org/elasticsearch/percolator/PercolateQueryBuilder.java

```diff
@@ -94,6 +95,7 @@
     private final String field;
     private final String documentType;
     private final BytesReference document;
+    private final XContentType xContentType;
```

Comment:
can you make it clear that it is the document xContentType? I got confused for a second.